examples/object_detection/main.py

```diff
@@ -305,7 +305,8 @@ def train(net, compression_ctrl, train_data_loader, test_data_loader, criterion,
 
         epoch = iteration // epoch_size
 
-        if (iteration + 1) % epoch_size == 0:
+        compression_ctrl.scheduler.step(iteration - config.start_iter)
+        if iteration % epoch_size == 0:
```

Comment:
Not all block should be under `if iteration % epoch_size == 0`, some should be under `if iteration + 1 % epoch_size == 0`

Reply:
Fixed.